docs/06_CONTEXT-MANAGER.md

```diff
@@ -624,4 +718,13 @@ sdk.contextManager().setSecureStorageImpl(new MyOwnSecureImplementation());
 ```
 </details>
 
+### Python
+<details>
+<summary>Show Details</summary>
+
+```csharp
+// TODO
```

Comment:
Missed this TODO or intentional?

Reply:
Intentionally, this cannot be used with Python and C# right now.